pkgs/applications/editors/vscode/generic.nix

```diff
@@ -108,6 +106,10 @@ let
 
     preFixup = ''
       gappsWrapperArgs+=(
+        --set GTK_USE_PORTAL 1
+        ${ # we cannot use runtimeDependencies otherwise libdbusmenu do not work on kde
+          lib.optionalString stdenv.isLinux
+          "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ (lib.getLib systemd) fontconfig.lib libdbusmenu wayland ]}"}
```

Comment:
Why are you moving _all_ `runtimeDependencies` here? Shouldn't you only move `libdbusmenu` here?